### DISCUSSION TITLE
Reusing the Examples

### DISCUSSION THREAD

**Discussion by paulhdk** [HaishinKit/HaishinKit.swift#1461] [Q&A] [answered]
Hi!

I'd like to reuse some of the examples (visionOS, iOSSwiftUI) for a research project.
How do I correctly attribute HaishinKit?

From what I understand, the research project needs to have the same license as HashinKit, i.e. the same LICENSE.MD, correct?

Many thanks!

**Comment by shogo4405** [comment, marked as answer]
If you copy it as is, please handle it under the BSD 3-Clause License. However, since the Examples are sample code written for library users, feel free to use them as reference.
